### DISCUSSION TITLE
Windows Subsystem for Android (WSA) access to local network

### DISCUSSION THREAD

**Discussion by Wembly1** [microsoft/WSL#7936] [Q&A]
Hello All,

I installed preview version of Windows Subsystem for Android on Windows 11. It works fine and I can install apps. I would like to test Apps that require full access to local network of the host computer. For example I want to use Spotify App to control a smart speaker via Spotify Connected. For this the Spotify App within WSA need to access to the local network via multiple protocols, for example Bonjour and others.

I wonder whether somebody could give me a hint on this.

**Comment by philsward** [comment]
Welcome to the reason they're shutting it down. Nobody can even do anything with WSA from a usable standpoint, so nobody is using it. It's a gimmick. Local networking doesn't work, bluetooth doesn't work and literally nothing people "need" to make WSA work, actually works.

The experimental local networking option? Doesn't work.

Such a shame because this could have created some awesome opportunities for people.
